gatsby_fp/src/components/fieldComponents/intellCont.js

```diff
@@ -0,0 +1,195 @@
+import React from "react"
+
+export default function IntellCont ({intellContList}){
```

Comment:
We should probably name this component for what it represents, rather than what the data source calls it. Makes it easier to reason about in context.